### ISSUE TITLE
Sammler - Interaktive Elemente entsprechen nicht der erweiterten Größe

### ISSUE THREAD

**Issue by dlekim** [RRZE-Webteam/FAU-Elemental#379]
Interaktive Elemente wie Schaltflächen oder Links sollten mindestens 44 x 44 Pixel groß sein, damit sie von allen Benutzern problemlos angeklickt oder angetippt werden können.

Aktuelle Größe der Elemente: 22 - 40 px Höhe

--

Vorkommen:
- Breadcrumb
- Hauptnavigation (Direkteinstiege / Burger Menu Button)
- Footer Wichtige Links und die dazugehörigen Überschriften
- Secondary Footer Links
- Teasergrid 

<img width="1332" height="181" alt="Image" src="https://github.com/user-attachments/assets/4e36856e-fa76-49b9-ba0b-b9a2b2c3a6b3" /> 
Breadcrumb + Hauptnavigation

<img width="1345" height="131" alt="Image" src="https://github.com/user-attachments/assets/008d04f9-2101-44f7-a16f-72e9908cd572" />

<img width="644" height="220" alt="Image" src="https://github.com/user-attachments/assets/c8e3e7f2-358d-4d2c-887d-fc282fbc1313" />
Footernavigation

<img width="1363" height="173" alt="Image" src="https://github.com/user-attachments/assets/9590a5b2-cc22-45e7-8dbc-1effafeec3df" /> 
Portalmenü (nicht wundern, Bilder fehlen in der Siteimprove Ansicht gerade)

<img width="819" height="146" alt="Image" src="https://github.com/user-attachments/assets/a6d13c86-319e-4ab7-a3a6-efb1fa1f3c75" />
Bulletlist

--

Infos:
Eine angemessene Größe der interaktiven Elemente ist wichtig, damit auch Menschen mit körperlichen Beeinträchtigungen oder Menschen mit Touchscreens problemlos damit interagieren können.
– Siteimprove Alfa: [Technische Dokumentation für Regel SIA-R111 (auf Englisch)](https://alfa.siteimprove.com/rules/sia-r111) – WCAG 2.2: [Understanding Success Criterion 2.5.5: Target Size Enhanced](https://www.w3.org/WAI/WCAG22/Understanding/target-size-enhanced)

**Comment by danielboese** [comment]
@dlekim Bildnachweise und Social Icons passen wir an, bei den Listen greift meiner Kenntnis nach folgende Ausnahme:

https://www.w3.org/WAI/WCAG21/Understanding/target-size.html

"Inline: The success criterion does not apply to inline targets in sentences or blocks of text. Content is often designed to reflow based on the screen width available. This is known as responsive design and makes it easier to read since you do not need to scroll both horizontally and vertically. In reflowed content, the targets can appear anywhere on a line and can change position based on the width of the available screen. Since targets can appear anywhere on the line, the size cannot be larger than the available text and spacing between the sentences or paragraphs, otherwise the targets could overlap. It is for this reason targets which are contained within one or more sentences are excluded from the target size requirements."

Bei den Listen wäre dann das Problem, dass wir ja nie genau wissen, wie lang der Text eines Listenpunktes ist. Hier würde es dann spätestens bei den Mobile-Ansichten zu großen vertikalen Lücken kommen, wenn wir die Links auf 44px Höhe zwingen, da der Text mehrzeilig werden kann.

**Comment by dlekim** [comment]
@danielboese da hier auch die Ausnahme wie oben beschrieben greift, schließe ich dieses Ticket. 

Es wird in die Barrierefreiheitserklärung mit aufgenommen.
